packages/react-scripts/template/README.md

```diff
@@ -346,6 +346,35 @@ Learn more about ES6 modules:
 * [Exploring ES6: Modules](http://exploringjs.com/es6/ch_modules.html)
 * [Understanding ES6: Modules](https://leanpub.com/understandinges6/read#leanpub-auto-encapsulating-code-with-modules)
 
+## Importing a Component using path relative to src (Absolute path)
+
+Create a file called `.env` in the root of the project, then add the following:
```

Comment:
>I believe there was an argument posted somewhere on these issues arguing to check .env into source control, as it doesn't contain any secrets (client side code shouldn't).

Yes, we'll remove it from gitignore as part of https://github.com/facebookincubator/create-react-app/pull/1344.